ext/pcntl/pcntl.c

```diff
@@ -1515,11 +1509,24 @@ PHP_FUNCTION(pcntl_rfork)
 
   /* This is a flag to use with great caution in general, preferably not within PHP */
   if ((flags & RFMEM) != 0) {
-    flags &= ~RFMEM;
+    zend_argument_value_error(1, "must not include RFMEM value, not allowed within this context");
+    RETURN_THROWS();
+  }
+
+  if ((flags & RFSIGSHARE) != 0) {
+    zend_argument_value_error(1, "must not include RFSIGSHARE value, not allowed within this context");
```

Comment:
Isn't it the indentation broken in the whole function?

Reply:
Oh yeah, looks like it's indented with 2 spaces rather than tabs. Hard to see when it's consistently wrong ^^